tests/test_modeling_common.py

```diff
@@ -3582,6 +3582,8 @@ def get_mean_reldiff(failcase, x, ref, atol, rtol):
 
         for model_class in self.all_model_classes:
             config, inputs_dict = self.model_tester.prepare_config_and_inputs_for_common()
+            if hasattr(config, "vision_config"):
+                config.vision_config._attn_implementation = config._attn_implementation
```

Comment:
Same here - we shouldn't need to set this to make the test work

Reply:
Done in e78108b4c.